### DISCUSSION TITLE
Multiple series in area chart with time primary axis

### DISCUSSION THREAD

**Discussion by aaafolabi** [TanStack/react-charts#227] [Q&A]
Hi all,

Is it possible to stack a number of series of varying lengths on a time axis?
For example:
series A: [{x: 01/01/2021, y: 2}, {x: 01/02/2021, y: 3}, {x: 01/03/2021, y: 5}, {x: 01/04/2021, y: 10}, {x: 01/05/2021, y: 12}]
series B: [{x: 01/01/2021, y: 3}, {x: 01/03/2021, y: 7}, {x: 01/05/2021, y: 9}]

Thanks

**Comment by tannerlinsley** [comment]
Should work fine

**Reply by aaafolabi** [reply]
I'm seeing errors from buildAxis.linear where its expecting the series to be of equal length.

**Comment by tannerlinsley** [comment]
Is that error coming from d3? If so, then you’ll need to fill in with either undefined or null data points.

**Reply by aaafolabi** [reply]
It appears to come from _react-charts/src/utils/buildAxis.linear.js /_ . There's a _stackSeries_ function in there that's throwing a null reference error.

**Reply by tannerlinsley** [reply]
Codesandbox I can look at?

**Reply by aaafolabi** [reply]
I have created a sample here: https://codesandbox.io/embed/compassionate-mountain-6vxq5?fontsize=14&hidenavigation=1&theme=dark

**Reply by tannerlinsley** [reply]
looks to be invalid Json

**Reply by aaafolabi** [reply]
I have fixed all the json errors I noticed. Can you point me in the direction of what's wrong please?

**Reply by tannerlinsley** [reply]
Still getting an issue in that sandbox: 
![9582A90F-FC7E-45CB-8764-B51C8EA11CE9](https://user-images.githubusercontent.com/5580297/126918461-086df769-fc18-480c-8704-52918863bd6d.png)

**Reply by aaafolabi** [reply]
Sorry, had unsaved changes in the json. Please try again now.

**Reply by tannerlinsley** [reply]
The latest release should handle this use case now: https://codesandbox.io/s/prod-bush-mve0h?file=/package.json. The package is still releasing as of this second, so give it minute if you read this quickly.
